modules/admin/proc-configuring-tls-with-postgresql.adoc

```diff
@@ -0,0 +1,154 @@
+[id="proc-configuring-tls-with-postgresql_{context}"]
+= Configuring TLS with the PostgreSQL database in {product-very-short}
+
+You can also configure Transport Layer Security (TLS) with the PostgreSQL database for secure transfer of data over different communication channels. 
+
+.Prerequisites
+
+* You have installed the CrunchyData PostgreSQL Operator on the {ocp-short} cluster. For more information, see link:https://access.crunchydata.com/documentation/postgres-operator/latest/quickstart#installation[Installing the PostgreSQL Operator].
+
+* You have created the PostgreSQL secret named `postgress-external-db-cluster-cert` in the namespace where you plan to configure the PostgreSQL database. For more information, see link:{LinkAdminGuide}#proc-creating-secrets-for-postgresql_admin-rhdh[Creating PostgreSQL secrets].  
+
+.Procedure
+
+. Configure the PostgreSQL database in a namespace by using the following YAML code:
++
+[source,yaml]
+----
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
```

Comment:
Do we need to suggest how to install Crunchy DB here?
We do not put this kind of info for RDS and Azure (it clearly was out of scope).

Reply:
I agree, @gazarenkov. I think we should leave it to customers how would they configure a  `PostgresCluster` resource based on their needs. We need not provide any handhold over here.